### PR TITLE
Revert: incouding atomic sites with Jetpack

### PR DESCRIPTION
**Pull request by leandroalonso**
Test on: https://github.com/wordpress-mobile/WordPress-iOS/pull/16613

### CODE REVIEW

File: WordPressKit.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "4.34.0"
+  s.version       = "4.34.1"
```
Comment (momo-ozawa) [inline WordPressKit.podspec:3]: Should this be 4.34.1-beta.1?

**Review by momo-ozawa — APPROVED** [review]
LGTM w/ a question about the version.

File: WordPressKit.podspec (hunk 1 above)
Comment (leandroalonso) [reply WordPressKit.podspec:3]: Actually no. Since I'll be targeting the `release` branch on WPiOS this should be a hotfix. :)

Comment (momo-ozawa) [reply WordPressKit.podspec:3]: Gotcha. Thanks for the clarification!

**Pull request merged**